src/processor/operator/copy/copy_rel.cpp

```diff
@@ -11,8 +11,9 @@ uint64_t CopyRel::executeInternal(
     kuzu::common::TaskScheduler* taskScheduler, ExecutionContext* executionContext) {
     auto relCopier = make_unique<RelCopyExecutor>(copyDescription, wal->getDirectory(),
         *taskScheduler, *catalog, nodesStore, table, relsStatistics);
-    auto numRelsCopied = relCopier->copy(executionContext);
     wal->logCopyRelRecord(tableID);
+    wal->flushAllPages();
```

Comment:
As discussed, since we currently haven't implemented the undo logic, i think you can remove this function call.